.github/workflows/unit-tests.yml

```diff
@@ -117,3 +123,33 @@ jobs:
           name: pytest-warning-report-html
           path: |
             reports/pytest_warnings/warning_report_all.html
+
+  # Combine and upload coverage reports.
+  coverage:
+    needs: run-tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.8 ]
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Setup Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Download all artifacts
+        uses: actions/download-artifact@v2
+
+      - name: Install Python dependencies
+        run: |
+          pip install -r requirements/edx/coverage.txt
+
+      - name: Run coverage
+        run: |
+          coverage combine /home/runner/work/edx-platform/edx-platform/coverage*
```

Comment:
I don't think `coverage*` is the right pattern for the data files to combine.  Better to leave off the file specification and let coverage find the data files based on its configuration.